doc/user/caching.xml

```diff
@@ -534,22 +513,25 @@ Program('prog', ['f1.c', 'f2.c', 'f3.c', 'f4.c', 'f5.c'])
 
     <para>
 
-    &SCons;' internal <classname>CacheDir</classname> class can be extended to support customization
-    around the details of caching behaviors, for example using compressed cache files,
-    encrypted cache files, gathering statistics and data, or many other aspects.
+    You can customize the behavior of derived-file caching to
+    add your own features, for example to support compressed and/or
+    encrypted cache files, modify cache file permissions to better
+    support shared caches, gather additional statistics and data, etc.
 
     </para>
 
     <para>
 
-    To create your own custom cache class,
-    your custom class must be a subclass
-    of the <classname>SCons.CacheDir.CacheDir</classname> class.
-    You can then pass your custom class to the &f-link-CacheDir;
-    method or set the &consvar;
-    &cv-link-CACHEDIR_CLASS; to the class before configuring the cache
-    in that environment.
-    SCons will internally invoke and use your custom class when performing
+    To define custom cache behavior, subclass the
+    <classname>SCons.CacheDir.CacheDir</classname> class,
+    specializing those methods you want to change.
+    You can pass this custom class as the <parameter>custom_class</parameter>
+    paramter when calling &f-link-CacheDir; for global reach,
```

Comment:
typo: paramter -> parameter

Reply:
updated.  I seem to make that specific typo a lot.